src/components/WalletBalance/index.tsx

```diff
@@ -1,51 +1,54 @@
-import React, {Fragment, useEffect, useState} from 'react'
+import React from 'react'
 import { ChainId, TokenAmount } from '@sushiswap/sdk'
-import { AdditionalChainId } from '../../constants'
-import { CustomLightSpinner } from 'theme'
-import Circle from 'assets/images/blue-loader.svg'
+import { Fragment, useEffect, useState } from 'react'
 import { LoadingPlaceHolder } from 'theme/components'
+import { AdditionalChainId } from '../../constants'
 
 export interface Balances {
-  G$: TokenAmount | undefined,
-  GDX: TokenAmount | undefined,
-  GOOD: TokenAmount | undefined
+    G$: TokenAmount | undefined
+    GDX: TokenAmount | undefined
+    GOOD: TokenAmount | undefined
+}
+
+export interface BalancesV2 {
+    G$: any
+    GOOD: any
 }
 
 export type WalletBalanceProps = {
-  balances: Balances,
-  chainId: ChainId,
+    balances: BalancesV2
+    chainId: ChainId
 }
```

Comment:
get type from the sdk (preferably) or from usedapp

Reply:
Moved to sdk